weed/util/http_header.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Move this file to s3api folder

Reply:
And remove all unused variables.

Reply:
it seems will cause "import cycle" if move http_header to s3api.